components/tfx/Evaluator/with_URI_IO/component.py

```diff
@@ -0,0 +1,103 @@
+# flake8: noqa
+
+from typing import NamedTuple
+
+def Evaluator(
+    examples_uri: 'ExamplesUri',
+    model_uri: 'ModelUri',
+    output_evaluation_uri: 'ModelEvaluationUri',
+    #output_blessing_uri: 'ModelBlessingUri',
+    baseline_model_uri: 'ModelUri' = None,
+    schema_uri: 'SchemaUri' = None,
+    eval_config: {'JsonObject': {'data_type': 'proto:tensorflow_model_analysis.EvalConfig'}} = None,
+    feature_slicing_spec: {'JsonObject': {'data_type': 'proto:tfx.components.evaluator.FeatureSlicingSpec'}} = None,
+    fairness_indicator_thresholds: list = None,
+    beam_pipeline_args: list = None,
+) -> NamedTuple('Outputs', [
+    ('evaluation_uri', 'ModelEvaluationUri'),
+    #('blessing_uri', 'ModelBlessingUri'),
+]):
+    from tfx.components import Evaluator as component_class
+
+    #Generated code
+    import json
+    import os
+    import tempfile
+    import tensorflow
+    import uuid
+    from google.protobuf import json_format, message
+    from tfx.types import channel_utils, artifact_utils
+    from tfx.components.base import base_executor
+
+    arguments = locals().copy()
+
+    component_class_args = {}
+
+    for name, execution_parameter in component_class.SPEC_CLASS.PARAMETERS.items():
+        argument_value_obj = argument_value = arguments.get(name, None)
```

Comment:
Do we really need to assign `argument_value_obj` here?

Reply:
I agree - it looks confusing. Moved the assignment to the new else clause.